ambassador/ambassador/ir/irauth.py

```diff
@@ -141,22 +127,14 @@ def _load_auth(self, module: Resource):
                     self[key] = value
 
             self.referenced_by(module)
+        
+        self.to_header_list("allowed_request_headers", module)
+        self.to_header_list("allowed_authorization_headers", module)
 
-        headers = module.get('allowed_headers', None)
-
-        if headers:
-            allowed_headers = self.get('allowed_headers', [])
-
-            for hdr in headers:
-                if hdr not in allowed_headers:
-                    allowed_headers.append(hdr)
-
-            self['allowed_headers'] = allowed_headers
+        self["timeout_ms"] = module.get("timeout_ms", "3s")
```

Comment:
What happens if it's something like "3000"? after all, it's called "timeout_ms" -- should we just call it "timeout" and document it as a duration?

Reply:
Good point, as I mentioned in the comment above, I think we need to support both `timeout_ms` as `v0` an `timeout` in `v1`. Not entirely sure.

Reply:
Noted above, but since we use `timeout_ms` elsewhere in the Ambassador config objects, let's stick with `timeout_ms` for the aconf side of things.

Reply:
👍